src/components/m-dropdown/mixins/dropdown.scss

```diff
@@ -74,7 +74,6 @@
   &:focus {
     border: none;
     box-shadow: none;
-    outline: none;
```

Comment:
@jackblackCH 
@LucaMele 
`outline` is removed by intention here and replaced below with `color`.
Please undo.
Originally this was reviewed and approved with Anja Wagner, since she left axa. @jojackpot  or @kenaxa could be asked.

Reply:
as said in my privious comment above either focusable or ask ken or jonas

Reply:
Okay.

Please let me mention again, that all `outline` are replaced by `:focus`.
Concretely about this file check a few lines below please:

https://github.com/axa-ch/patterns-library/blob/554e86c369a59ecd21de571d1e1b018bc808d55f/src/components/m-dropdown/mixins/dropdown.scss#L80-L86

Reply:
@AndyOGo agree and needs to be fixed

Reply:
I re-added... Strongly disagree though. Just changing color is not enough, it decreases accessibility a lot for a little visual "issue". Disable people and/or keyboard users are used to an outline and a similar behavior as global as possible, at least page wide. I open this discussion with @jojackpot and @kenaxa .